starknet_py/net/http_client.py

```diff
@@ -63,10 +63,12 @@ async def call(self, method_name: str, params: dict):
         payload = {
             "jsonrpc": "2.0",
             "method": f"starknet_{method_name}",
-            "params": params,
             "id": 0,
         }
 
+        if params:
+            payload["params"] = params
+
```

Comment:
Not sure about this change. Yes, it can be omitted from the json-rpc point of view. 
In the starknet spec, however, if a method doesn't require parameters, an empty array is explicitly included in the "params" field (I wasn't able to find an exception):
```json
"params": [],
```
And here, if the params dict is empty, nothing will be passed. Maybe nodes accept that currently, but I think it would be better to pass empty array if params dict is empty just in case, wdyt?

Reply:
Thanks! Sounds good to me.